tests/containers/workbenches/workbench_image_test.py

```diff
@@ -108,6 +108,18 @@ def test_ipv6_only(self, subtests: pytest_subtests.SubTests, image: str, test_fr
         finally:
             docker_utils.NotebookContainer(container).stop(timeout=0)
 
+    @pytest.mark.openshift
+    def test_image_run_on_openshift(self, image: str):
+        skip_if_not_workbench_image(image)
```

Comment:
probably will need to be adjusted per the latest change #918

Reply:
done